frontend/views/containers/modals/CreateGroup.vue

```diff
@@ -1,48 +1,53 @@
 <template lang='pug'>
-//- TODO: move this into modal
-main.main#create-group-page
-  .steps
-    router-link.step(
+modal-base-template
+  .steps(v-if='currentStep + 1 < config.steps.length')
+    button.step(
       v-for='(step, index) in config.steps'
       :key='index'
-      :to='{name: step}'
       :class='[currentStep === index ? "active" : "", currentStep < index ? "next" : ""]'
+      @click='redirect(step)'
+      v-if='step != "GroupWelcome"'
     ) {{ index + 1 }}
 
+  .wrapper.mobile-steps.subtitle(v-if='currentStep + 1 < config.steps.length')
+    i18n(:args='{ current: currentStep + 1, max: config.steps.length - 1}') Step {current} of {max}
+
   transition(name='fade' mode='out-in')
-    router-view(
+    component(
+      :is='content'
       :group='form'
       :v='$v.form'
       @next='next'
       @focusref='focusRef'
       @input='payload => updateGroupData(payload)'
     )
+      .buttons(v-if='currentStep + 1 < config.steps.length')
+        i18n(
+          class='is-outlined'
+          tag='button'
+          @click='prev'
+          :args='{ text: currentStep === 0 ? "Cancel" : "Back" }'
+          data-test='prevBtn'
+        ) {text}
```

Comment:
The rule is that *string literals must be encased in `i18n` or `L`*.

This use of `i18n` circumvents the rule because the string literals `"Cancel"` and `"Back"` are not wrapped.

Therefore this would produce invalid output.

There are several ways this can be fixed, but probably the simplest (and therefore most correct) here would be to avoid using `i18n`.

```pug
button.is-outlined(
  @click='prev'
  data-test='prevBtn'
) {{ currentStep === 0 ? L('Cancel') : L('Back') }}
```

Reply:
That's what I had before..

Reply:
I recall that before you had something similar, but it did not use the `L` function to wrap the string literals.